samples/TagHelperSample.Web/Views/Home/Create.cshtml

```diff
@@ -21,7 +21,7 @@
         @* - should helper remove the <div/> if list isn't generated? *@
         @* - (Html.ValidationSummary returns empty string despite non-empty message parameter) *@
         @* Acceptable values are: "None", "ModelOnly" and "All" *@
-        <div asp-validation-summary="ValidationSummary.All" style="color:blue" id="validation_day" class="form-group">
+        <div asp-validation-summary="@ValidationSummary.All" style="color:blue" id="validation_day" class="form-group">
```

Comment:
This is the sample site, there's another page in the site that has the non `@` variation. Wanted to showcase both ways.